source-academy/frontend
# Transfer groundControl from staff to admin route

### Description
Addresses #3031 by re-routing ground control from a staff to an admin route.
P.S.: still figuring out the backend. Will further changes be required?



### Type of change


### How to test

- Enter the test environment as an admin, and see that you are able to enter ground control normally.
- Copy the ground control route (should be '/groundcontrol')
- Switch the test environment to be a staff
- Check that the ground control button is now missing.
- Paste the ground control route. A 404 route should be expected.

- Go to `src/commons/navigationBar/AcademyNavigationBar` and change the ground control route into a staff route.
- Log in as staff. Manually append '/groundcontrol' to the URL to get to the ground control screen.
- Attempt to delete an assessment, or mass publish an assessment through the ground control panel.
- A 403 Forbidden error should show up.
- Other functions, such as unpublishing or publishing of a single submission in the gradings table, should still be possible.

### Checklist



- [X] I have tested this code - I have manually tested the website to check if I broke any frontend-backend links.
